components/service/glean/src/main/java/mozilla/components/service/glean/utils/DateUtils.kt

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.glean.utils
+
+import java.lang.StringBuilder
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.Locale
+
+internal const val DATE_FORMAT_PATTERN = "yyyy-MM-dd'T'HH:mmZ"
+
+/**
+ * Generate an ISO8601 compliant time string for the current time.
+ *
+ * @param date the [Date] object to convert to string
+ * @return a string containing the date and time.
+ */
+internal fun getISOTimeString(date: Date): String {
+    val dateFormat = SimpleDateFormat(DATE_FORMAT_PATTERN, Locale.US)
+    val timeString = StringBuilder(dateFormat.format(Date()))
```

Comment:
You aren't actually using the input parameter `date: Date` here and I think you probably meant to use it rather than `Date()`